benchmarks/requirements.txt

```diff
@@ -0,0 +1,4 @@
+# Dependencies needed to run the benchmarks
+
+# asv>0.5.1,<=0.6
```

Comment:
ASV started creating proper releases again a few days ago.
We should just use the PyPI versions.

Reply:
I agree. Is there a reason this is just commented out for now? I suggest we make the change in this PR.

Reply:
I thought there might be an issue with the benchmark runner, but I agree, we should probably make this change now

Reply:
The reason this was pinned was, if I remember correctly, that running the benchmarks quickly in dev mode was not available for the release. This made it hard to include the benchmarks into the CI.

Reply:
Ok, I added the PyPI version